tests/test_cudnn_compatible_gru.py

```diff
@@ -0,0 +1,481 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
these test functions are same with gru's except the cell type, will it be better to merge with them? otherwise, if we want to add more test cases, them we have to add 3 function.

Reply:
it's doable like contrib op test, but I suggest to keep it separated in this PR.

Reply:
got it.